backend/pkg/api/handle_topics_integration_test.go

```diff
@@ -376,16 +297,18 @@ func (s *APIIntegrationTestSuite) TestHandleGetTopics() {
 		assert.Equal(testutil.TopicNameForTest("get_topics_2"), getRes.Topics[2].TopicName)
 
 		// verify warning logs
-		allLogs := obs.All()
-
-		require.Len(allLogs, 1)
-		assert.Equal("config resource response has an error", allLogs[0].Message)
-		assert.Equal(zap.WarnLevel, allLogs[0].Level)
-		contextMap := allLogs[0].ContextMap()
-		logObj, ok := contextMap["error"]
-		assert.True(ok)
-		fieldValue, ok := logObj.(string)
-		assert.True(ok)
-		assert.Equal("INVALID_TOPIC_EXCEPTION: The request attempted to perform an operation on an invalid topic.", fieldValue)
+		/*
+			allLogs := obs.All()
+
+			require.Len(allLogs, 1)
+			assert.Equal("config resource response has an error", allLogs[0].Message)
+			assert.Equal(zap.WarnLevel, allLogs[0].Level)
+			contextMap := allLogs[0].ContextMap()
+			logObj, ok := contextMap["error"]
+			assert.True(ok)
+			fieldValue, ok := logObj.(string)
+			assert.True(ok)
+			assert.Equal("INVALID_TOPIC_EXCEPTION: The request attempted to perform an operation on an invalid topic.", fieldValue)
+		*/
```

Comment:
If there are new log assertions to verify behaviour lets add them?
If the behaviour is different now that there are no meaningful log assertions lets remove this commented out code.